docs/list.md

```diff
@@ -355,6 +355,19 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **Issues**: [Eligible projects and issues](https://github.com/search?q=org%3Aturbot+topic%3Ahacktoberfest&type=repositories).
 - **Notes**: Read more details in the [announcement post](https://turbot.com/blog/2023/10/hacktoberfest-2023) and join our [Turbot community](https://turbot.com/community/join) `#contributors` channel with any questions.
 
+### U
+
+#### **Uno Platform**
+
+- **Swag**: Stickers, Sweater, T-shirt, stickers and WinUI 3 ebook.
+- **Requirements**:
+    - Any open issue on the [Uno Platform GitHub](https://github.com/unoplatform/uno/blob/master/CONTRIBUTING.md) qualifies.
```

Comment:
This can be removed 
```suggestion
```

And this can be added below *How to Sign up*

```md
- **Issues**: [Curated Issues Board](https://github.com/orgs/unoplatform/projects/77/views/1) or you can request the addition of `hacktoberest` label on any [open issue](https://github.com/search?q=org%3Aunoplatform+is%3Aissue+is%3Aopen&type=issues) in any of the the Uno platform repos.
```